pkg/operator/ceph/cluster/cluster.go

```diff
@@ -78,6 +78,21 @@ func newCluster(c *cephv1.CephCluster, context *clusterd.Context) *cluster {
 
 func (c *cluster) detectCephMajorVersion(image string, timeout time.Duration) (string, error) {
 	// get the major ceph version by running "ceph --version" in the ceph image
+	podSpec := v1.PodSpec{
+		Containers: []v1.Container{
+			{
+				Command: []string{"ceph"},
+				Args:    []string{"--version"},
+				Name:    "version",
+				Image:   image,
+			},
+		},
+		RestartPolicy: v1.RestartPolicyOnFailure,
+	}
+
+	// apply "all" placement
```

Comment:
I'm not sure if `All` placement is the key we want to use here, though I don't really have an alternative.

Besides that, could you please document this behavior on the Ceph Cluster CRD doc page.

Reply:
Helps if I check the right directory... updated what I believe is the right file and force pushed the update. PTAL.